### ISSUE TITLE
Getting Ads, but NOT Receiving BAT since reset on 8/31/20.

### ISSUE THREAD

**Issue by X22396** [brave/brave-browser#11556]
## Description 

Since the reset on 8/31/20, I am receiving ads still, according to the ads list in Linux, I have recieved over 20 since the reset, however I ended 8/31/20 with 1.995 BAT, after recieving these 20+ ads, i am only at 2.02 BAT.

## Steps to Reproduce

   1. Watch Ad appear, or click on Sponsered Tab image
   2. Either Click on Ad, or dismiss ad
   3. BAT total does not increase

## Expected result:
BAT increase for each ad, Sponsored ad page

## Reproduces how often: 

Every Ad.

## Brave version (brave://version info)

Brave | 1.15.27 Chromium: 85.0.4183.83 (Official Build) nightly (64-bit)
-- | --
Revision | 94abc2237ae0c9a4cb5f035431c8adfb94324633-refs/branch-heads/4183@{#1658}
OS | Linux

NOTE: This started after Nightly update on 8/31/20, I have updated again on 09/01/20, but same issue.

## Version/Channel Information:

- Can you reproduce this issue with the current release?  Not Sure
- Can you reproduce this issue with the beta channel?     Not Sure
- Can you reproduce this issue with the dev channel?      Not Sure
- Can you reproduce this issue with the nightly channel?   YES, 

## Other Additional Information:

- Does the issue resolve itself when disabling Brave Shields?
- Does the issue resolve itself when disabling Brave Rewards? No, I turned off rewards and turned back on, no change.
- Is the issue reproducible on the latest version of Chrome?

**Comment by X22396** [comment]
Since my original post, I installed Brave Release version on same Linux Laptop. I am getting ads pop up, and they are counting toward my BAT, (although it started at 0). I am still not getting anything in the Nightly version.  

Also, now Sponsered Images are not showing up in either version. (Every 4th Tab is supposed to be sponsered ad, I have opened 8 tabs, in both versions (testing only, not normally do this), and no sponsered images shown. 

Not sure if installing new version reset ads algorythim, or if its just an issue with Nightly. 

thanks for your help.
